DESCRIPTION

```diff
@@ -1,13 +1,12 @@
 Package: jaspLearnBayes
 Type: Package
 Title: Learn Bayes Module for JASP
-Version: 0.15
-Date: 2020-11-21
+Version: 0.1.5
```

Comment:
Why the version change? Should not we stick to 0.15 for this release for this reason: https://github.com/jasp-stats/jasp-desktop/blob/development/Docs/development/jasp-upgrade-qml.md#migrating-from-oldschool-modules? @JorisGoosen

Reply:
Joris confirms it to be 0.15